pkg/communicator/websocketchannel.go

```diff
@@ -15,19 +15,24 @@
 package communicator
 
 import (
+	"context"
 	"errors"
+	"net/http"
+	"net/url"
 	"sync"
 	"time"
 
+	v4 "github.com/aws/aws-sdk-go-v2/aws/signer/v4"
```

Comment:
typo?

Reply:
the reason its used here is because the signer v4 package imports functions that look like `NewSigner`, but to distinguish that it comes from this module, you alias it so that when you call it, it looks like `v4.NewSigner`. Its really just an aesthetic choice, but my auto validator/linter/formatter made it look this way.